src/shared/generator.c

```diff
@@ -167,7 +167,9 @@ int generator_write_timeouts(
         usec_t u;
         int r;
 
-        r = fstab_filter_options(opts, "comment=systemd.device-timeout\0" "x-systemd.device-timeout\0",
+        r = fstab_filter_options(opts,
+                                 "comment=systemd.device-timeout\0" "x-systemd.device-timeout\0"
+                                 "comment=systemd.mount-timeout\0" "x-systemd.mount-timeout\0",
```

Comment:
Not sure what I did here... Looks like I misread the function. So we can completely drop this part?